text/stacks/0000-node-on-tiny.md

```diff
@@ -0,0 +1,53 @@
+# Add to Tiny Run Image to Enable Node Apps
+
+## Summary
+
+At the moment, Node.js applications are not compatible with the tiny builder. Therefor the `nodejs` buildpack does not support the tiny stack.
+
+The tiny build image is already capable of building the Node.js application.
+
+The tiny run image is missing `libstdc++6`and `libgcc1`.
+
+## Motivation
+
+To support running Node.js applications with the tiny builder.
+
+While not all Node.js applications could be supported by that, this could be a feasible approach:
+
+- `tiny`: Run Node.js applications **without** any native extenstions
+- `base`: Run Node.js applications **without** many native extensions
+- `full`: Run Node.js applications **with** many native extensions
+
+This should be desirable as some types of Node.js application would benefit of the reduced image size.
```

Comment:
Although this can only be a snapshot, I suppose it is worth pointing out the correctly known CVEs it would remove for Node apps / add for users of the tiny stack.

I guess this is the more crucial metric compared to image size.

Reply:
We looked at existing RFC and they were written like that. As you mentioned stating the CVE would only be a snapshot anyway.